cmd/tracetest/main.go

```diff
@@ -14,23 +18,44 @@ const (
 )
 
 func main() {
-	config, err := config.NewConfig(_configPath)
+	// tracer directory: /opt/macfe/traces
+	// log directory: /opt/macfe/traces/logs
+	conf, err := config.NewConfig(_configPath)
 	if err != nil {
-		panic("error reading config file")
+		panic("error reading conf file")
 	}
 
 	ctx := context.Background()
 
 	cfg := zap.NewDevelopmentConfig()
+	formattedTime := time.Now().Format("2006.01.02_15.04.05")
+	fileName := fmt.Sprintf("/opt/macfe/traces/logs/file_%s.log", formattedTime)
+	cfg.OutputPaths = []string{fileName}
 	logger, err := cfg.Build()
+	if err != nil {
+		panic(err)
+	}
+
+	conn, err := socketcan.DialContext(context.Background(), "can", "can0")
+	if err != nil {
+		panic(err)
+	}
+
+	client := canlink.NewCANClient(CANBMScan.Messages(), conn)
+	tx := socketcan.NewTransmitter(conn)
+
+	m := canlink.NewMcap(logger, client, conf.TracerDirectory, conf.BusName)
+	a := canlink.NewAsc(logger, conf.TracerDirectory, conf.BusName)
 
 	tracer := canlink.NewTracer(
-		config.CanInterface,
-		config.TracerDirectory,
+		conf.CanInterface,
+		conf.TracerDirectory,
 		logger,
-		canlink.WithBusName(config.BusName),
-		canlink.WithTimeout(3*time.Second))
+		conn,
+		canlink.WithBusName(conf.BusName),
```

Comment:
How are you going to handle more than one bus? This config does not make sense.

Reply:
wdym? we aren't going to handle more than 1 bus since each dbc is 1:1 with a bus

Reply:
make bus name const call it VEH_CAN